### ISSUE TITLE
Buttons

### ISSUE THREAD

**Issue by AshM10** [elsalvadordali/green-smile#18]
- [ ] Logout Button
- [ ] Previous Button
- [ ] Next Button

**Comment by mnosov622** [comment]
Hi !

Do you want to add these buttons ? I would love to work on this issue

**Comment by elsalvadordali** [comment]
Yeah we would love for you to contribute! Let me know if you need anything.
